src/status_im/contexts/onboarding/common/background/view.cljs

```diff
@@ -65,10 +65,7 @@
 
     (rn/use-effect
      (fn []
-       (reanimated/set-shared-value @paused? (not= view-id :screen/onboarding.intro))
-       (fn []
-         (when (= view-id :screen/onboarding.generating-keys)
-           (carousel.animation/cleanup-animation progress paused?))))
```

Comment:
As far as I know, this change only resets the carousel animation for the key generation screen. If it also cleans up the background animation (and is required), I can add it to the preparing-status screen.

Reply:
Okay, I’ve just re-added it—no point risking another bug.